src/ui/colors.ts

```diff
@@ -0,0 +1,68 @@
+/* Updated at Tue, 08 Feb 2022 23:20:31 GMT*/
+export const colors = {
+  /** Blue  */
+  'Blue/1': '#f6fafe',
+  'Blue/12': '#2c91ee',
+  'Blue/16': '#0f6ec5',
+  'Blue/20': '#0b4f8e',
+  'Blue/24': '#062b4c',
+  'Blue/4': '#c2dffa',
+  'Blue/8': '#72b6f4',
+  /** Green  */
+  'Green/1': '#e5f7dd',
+  'Green/12': '#57b62b',
+  'Green/16': '#469222',
+  'Green/20': '#356e1a',
+  'Green/24': '#234911',
+  'Green/4': '#bae9a3',
+  'Green/8': '#74d447',
+  /** Grey  */
+  'Grey/1|Background': '#fbfbfb',
```

Comment:
Background is an alias of Grey, also would it be better to have:
`Grey1`
`grey-1`
`grey1`
I like the last

Reply:
Sure, I'm missing some stuff that's why I still have it in WIP hehe, this was something that I wanted to ask you, so the idea will be to have `grey1` and `background` right?

Reply:
yes